### DISCUSSION TITLE
Option to Backup Files with strict root permissions?

### DISCUSSION THREAD

**Discussion by niki-on-github** [backube/volsync#692] [Q&A] [answered]
I have setup a `ReplicationSource` to minio bucket for gitea ([with the help of  this documentation page](https://volsync.readthedocs.io/en/v0.6.1/usage/restic/index.html)). The pod for the gitea backup always report an Error. Other backups work without problems. The log of the failed backup pod:

```
 Starting container
│ VolSync restic container version: v0.6.1+cd18f4e
│ backup
│ Testing mandatory env variables 
│ == Checking directory for content === 
│ == Initialize Dir ======= 
│ ID        Time                 Host        Tags        Paths
│ ------------------------------------------------------------
│ ce817b05  2023-03-22 17:22:29  volsync                 /data
│ e380192f  2023-03-22 17:32:43  volsync                 /data
│ ------------------------------------------------------------ 
│ 2 snapshots      
│ === Starting backup ===
│ /data / 
│ using parent snapshot b900f845
│ [0:00] 0 files 0 B, total 0 files 0 B, 0 errors 
│ error: Open: open git/.ssh: permission denied
│ error: Open: open indexers: permission denied
│ error: open jwt/private.pem: permission denied
│
│ Files:           0 new,     0 changed,    15 unmodified 
│ Dirs:            0 new,     0 changed,    18 unmodified
│ Added to the repo: 0 B
│
│ processed 15 files, 7.188 KiB in 0:00
│ snapshot 99334931 saved 
│ Warning: at least one source file could not be read  
```

The file `jwt/private.pem` has the access level `-rw-------` for uid `0` and gid `0`. The folder `git/.ssh` is `drwx------` for `uid=0,gid=0`.

Is there a configuration option I can set to snapshot these files as well?

**Comment by tesshuflower** [comment, marked as answer]
@niki-on-github I think the docs for 0.6.x don't have the latest when it comes to restic mover permissions - but these instructions apply:  https://volsync.readthedocs.io/en/latest/usage/permissionmodel.html

Essentially restic is running as non-root by default in VolSync v0.6.1.  If you want to run it as root, you can opt-in by setting the following annotation on the namespace where you create your replicationsource/replicationdestination:

```
volsync.backube/privileged-movers: "true"
```

**Reply by niki-on-github** [reply]
With this option it works now - Thank you
